src/Components/WatchLater/WatchLater.jsx

```diff
@@ -1,7 +1,41 @@
 import React from "react";
+import { useVideo } from "../../Context/VideoContext";
```

Comment:
you can create an index.js inside contexts folder and import all the contexts from there!

Reply:
Thank you for the suggestion. But I only have 2 contexts so I don't think there's a need to create a separate index file just for 2 files.